pytripgui/app_logic/app_callbacks.py

```diff
@@ -216,6 +219,41 @@ def open_dicom_callback(self, patient_item):
         self.app_model.viewcanvases.set_patient(patient)
         return True
 
+    def export_patient_voxelplan_callback(self, patient_item):
+        # exports patient cube to Voxelplan format (.hed, .ctx, .vdx) with the selected name
+        logger.debug("Voxelplan export start.")
+        path = self.parent_gui.save_file_path("Export patient to Voxelplan", "Voxelplan (*.hed)")
+
+        if path == "":
+            return False
+
+        path_base, extension = os.path.splitext(path)
+        path, basename = os.path.split(path_base)
+        logger.info("Voxelplan export to: " + path + " with plan basename: " + basename)
+
+        patient_item.data.ctx.write(os.path.join(path, basename + ".ctx"))
+        patient_item.data.vdx.write(os.path.join(path, basename + ".vdx"))
```

Comment:
Same as above:
https://github.com/pytrip/pytrip/blob/master/pytrip/vdx.py#L93

Reply:
What will happen if VDX data is not present ?

Reply:
VDX import is now skipped if it doesn't exist.